src/components/networked-video-source-green-screen.js

```diff
@@ -0,0 +1,164 @@
+
+    NAF.utils.getNetworkedEntity(this.el).then((networkedEl) => {
+      const ownerId = networkedEl.components.networked.data.owner;
+      if (ownerId) {
+        NAF.connection.adapter.getMediaStream(ownerId, this.data.streamName)
+          .then(this._setMediaStream)
+          .catch((e) => naf.log.error(`Error getting media stream for ${ownerId}`, e));
+      } else {
+        // Correctly configured local entity, perhaps do something here for enabling debug audio loopback
+      }
+    });
+  },
+
+  _setMediaStream(newStream) {
+
+    if(!this.video) {
+      this.setupVideo();
+    }
+
+    if(newStream != this.stream) {
+
+      if (this.stream) {
+        this._clearMediaStream();
+      }
+
+      if (newStream) {
+        this.video.srcObject = newStream;
+
+        const playResult = this.video.play();
+        if (playResult instanceof Promise) {
+          playResult.catch((e) => naf.log.error(`Error play video stream`, e));
+        }
+
+        if (this.videoTexture) {
+          this.videoTexture.dispose();
+        }
+
+        this.videoTexture = new THREE.VideoTexture(this.video);
+        this.videoTexture.format = THREE.RGBAFormat;
+
+        const mesh = this.el.getObject3D('mesh');
+
+        //--------- begin replace green with transparent ---------
+
+        this.uniforms = {};
+
+        this.uniforms.uMap = {type: 't', value: this.videoTexture }
+        this.uniforms.GreenThresholdIn = {type: 'float', value: this.data.GreenThresholdIn};
+
+
+
+        this.uniforms = THREE.UniformsUtils.merge([
+                                                      this.uniforms,
+                                                      THREE.UniformsLib['lights']
+                                                    ]);
+
+        this.materialIncoming =  new THREE.ShaderMaterial({
+                                  uniforms: this.uniforms
+                                });
+
+        this.materialIncoming.vertexShader = `
+                         varying vec2 vUv;
+
+                        void main() {
+                            vec4 worldPosition = modelViewMatrix * vec4( position, 1.0 );
+                            vec3 vWorldPosition = worldPosition.xyz;
+                            vUv = uv;
+                            gl_Position = projectionMatrix * modelViewMatrix * vec4( position, 1.0 );
+                        }
+                      `;
+
+        this.materialIncoming.fragmentShader = `
+                           varying vec2 vUv;
+                           uniform sampler2D uMap;
+                           uniform float GreenThresholdIn;
+                           
+                           void main() {
+                                vec2 uv = vUv;
+                                vec4 tex1 = texture2D(uMap, uv * 1.0);
+                                 if (tex1.g - tex1.r > GreenThresholdIn)
+                                    gl_FragColor = vec4(0,0,0,0);
```

Comment:
Did you try discard like I said in https://github.com/networked-aframe/networked-aframe/issues/291#issuecomment-939309492 ?

Reply:
Today it worked with 
discard;